lib/database_transform/schema.rb

```diff
@@ -23,7 +23,25 @@ def self.transform_table(source_table, args = {}, &proc)
     transform.instance_eval(&proc) if proc
   end
 
+  delegate :around_job, :thread, to: :class
   class << self
+    # Store the block if a block is given, otherwise the stored block will be returned.
+    def around_job(&block)
+      if block_given?
+        @around_job = block
+      else
+        @around_job
+      end
+    end
+
+    def thread(number = nil)
+      if number
+        @thread = number
+      else
+        @thread
```

Comment:
I didn't handle the default value here, it will return nil by default.   Anyway you need to again give a default in schema_table, because schema can be nil.